server/src/game-implementations/types.ts

```diff
@@ -164,7 +164,8 @@ export type GPTGoalProgressFormatters<GPT extends GPTString> = {
  * indicating what the error was on failure.
  */
 export type ScoreValidator<GPT extends GPTString> = (
-	score: ScoreDocument<GPT>
```

Comment:
This is sensible and should be merged regardless of the sheet data issues, although `chart` need not be optional.

Reply:
I'll make a separate PR for that and set this one as draft until I can figure out another solution for stepcounts then